packages/fast-jss-manager-angular/src/manage-jss.ts

```diff
@@ -0,0 +1,295 @@
+import { Component, Input, ElementRef, SimpleChanges } from "@angular/core";
+import { SheetsManager, StyleSheet, getDynamicStyles } from "jss";
+import * as uuid from "uuid/v1";
+import getStaticStyles from "./utilities/get-static-styles";
+import { eventTypes } from "./utilities/get-event-types";
+import { isEmptyObject } from "./utilities/object";
+import jss, { stylesheetManager } from "./jss";
+
+/**
+ * Type definition for a function that resovles to a CSS property value
+ * It optionally expects a config object.
+ * TODO: How can we add typings to this config object
```

Comment:
Please file an issue for this Todo statement and add the number in the format of TODO: #123 : title